third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -270,6 +270,95 @@ convert_val_Fp8_to_Fp16(Location loc, ConversionPatternRewriter &rewriter,
   return ret;
 }
 
+static SmallVector<Value> cvtScaleFp8ToFp32(Location loc,
+                                            ConversionPatternRewriter &rewriter,
+                                            Value v0, Value v1,
+                                            const std::string &fp8_format) {
+
+  assert(fp8_format == "ocp_fp8" || fp8_format == "ocp_bf8");
```

Comment:
Can we avoid using strings here? You can use a templated function given the only difference is the ROCDL op being created?